sbt-plugin/src/sbt-test/linker/basic-dynamic-import/Main.scala

```diff
@@ -0,0 +1,12 @@
+package test
+
+import scala.scalajs.js
+
+object Main {
+  // Regression test for #4386. Arguably a bit of an abuse of sbt-test.
```

Comment:
Could we use a linker test instead? Or is it too hard to check that it actually works?

Reply:
I think we'd have to execute the JS we generate to be sure it works. So that would need ES module setup etc. :S Alternatively, we can also drop this test.

Reply:
I suggest we drop the test. I'm a bit reluctant to create a precedent of using scripted tests for things that have in fact nothing to do with the sbt plugin. In Scala Native those have proliferated, and it's a nightmare in terms of test suite execution time.